target/classes/application.yml

```diff
@@ -0,0 +1,29 @@
+management:
```

Comment:
Not sure how this has snuck in but I don't think target classes should be in the PR